lib/cc/analyzer/engine.rb

```diff
@@ -92,8 +92,14 @@ def docker_run_command
       end
 
       def env_file
+        contents = "ENGINE_CONFIG=#{@config_json}"
+
+        if contents.size > 64 * 1024
+          raise EngineFailure, "Config for engine #{name} exceeds 64k character limit"
```

Comment:
Do you think there's anything more valuable we can say here? Unfortunately, I'm having trouble thinking of a concrete improvement, but I find this message as-is a little lacking from the end-user's perspective.